cypress/integration/form-builder/conditionalFieldset.test.js

```diff
@@ -0,0 +1,384 @@
+  multiHiddenBooleanTrue1: 'Lorem',
+  multiHiddenBooleanTrue2: 'Lorem',
+  multiHiddenCallbackFalse1: 'Lorem',
+  multiHiddenCallbackFalse2: 'Lorem',
+  multiHiddenCallbackTrue1: 'Lorem',
+  multiHiddenCallbackTrue2: 'Lorem',
+  multiReadOnlyBooleanFalse1: 'Lorem',
+  multiReadOnlyBooleanFalse2: 'Lorem',
+  multiReadOnlyBooleanTrue1: 'Lorem',
+  multiReadOnlyBooleanTrue2: 'Lorem',
+  multiReadOnlyCallbackFalse1: 'Lorem',
+  multiReadOnlyCallbackFalse2: 'Lorem',
+  multiReadOnlyCallbackTrue1: 'Lorem',
+  multiReadOnlyCallbackTrue2: 'Lorem',
+  // Single
+  singleHiddenBooleanFalse1: 'Lorem',
+  singleHiddenBooleanTrue1: 'Lorem',
+  singleHiddenCallbackFalse1: 'Lorem',
+  singleHiddenCallbackTrue1: 'Lorem',
+  singleReadOnlyBooleanFalse1: 'Lorem',
+  singleReadOnlyBooleanTrue1: 'Lorem',
+  singleReadOnlyCallbackFalse1: 'Lorem',
+  singleReadOnlyCallbackTrue1: 'Lorem',
+}
+
+describe('@sanity/field: Multi fieldset and review changes', () => {
+  before(() => {
+    setUpSuite()
+  })
+
+  beforeEach(() => {
+    cy.viewport(2000, 3500)
+  })
+
+  // Hidden boolean false
+  it('when hidden property is false, the fieldset is rendered', () => {
+    cy.get('[data-testid="fieldset-multiHiddenBooleanFalse"]').should('be.visible')
+  })
+
+  //Hidden boolean true
+  it('when hidden property is true, the fieldset is not rendered', () => {
+    cy.get('[data-testid="fieldset-multiHiddenBooleanTrue"]').should('not.exist')
+  })
+
+  //Hidden callback (document) false
+  it('when the hidden callback (document) returns false, the fieldset is rendered', () => {
+    cy.get('[data-testid="fieldset-multiHiddenCallbackFalse"]').should('be.visible')
+  })
+
+  //Hidden callback (document) true
+  it('when the hidden callback (document) returns true, the fieldset is not rendered', () => {
+    cy.get('[data-testid="fieldset-multiHiddenCallbackTrue"]').should('not.exist')
+  })
+
+  //Read only boolean false
+  it('when the readOnly property is false, the fieldset is enabled', () => {
+    cy.get('[data-testid="fieldset-multiReadOnlyBooleanFalse"]')
+      .find('input')
+      .should('not.have.attr', 'readonly')
+  })
+
+  //Read only boolean true
+  it('when the readOnly property is true, the fieldset is disabled', () => {
+    cy.get('[data-testid="fieldset-multiReadOnlyBooleanTrue"]')
+      .find('input')
+      .should('have.attr', 'readonly')
+  })
+
+  //Read only callback (document) false
+  it('when the readOnly callback (document) returns false, the fieldset is enabled', () => {
+    cy.get('[data-testid="fieldset-multiReadOnlyCallbackFalse"]')
+      .find('input')
+      .should('not.have.attr', 'readonly')
+  })
+
+  //Read only callback (document) true
+  it('when the readOnly callback (document) returns true, the fieldset is disabled', () => {
+    cy.get('[data-testid="fieldset-multiReadOnlyCallbackTrue"]')
+      .find('input')
+      .should('have.attr', 'readonly')
+  })
+
+  // eslint-disable-next-line no-warning-comments
+  /* @TODO
```

Comment:
`it.todo` is unfortunately not something that cypress has available or at least I wasn't able to get it to work. 😞

Reply:
yeah that's a shame 😔